.github/workflows/release_to_stable.yaml

```diff
@@ -0,0 +1,41 @@
+name: Release all FINOS Legend charms to stable
+
+on:
+  workflow_dispatch:
+
+jobs:
+  release-to-stable:
+    name: Release FINOS Legend charms to stable
+    runs-on: ubuntu-latest
+    env:
+      CHARMCRAFT_AUTH: "${{ secrets.CHARMHUB_TOKEN }}"
+    steps:
+      - run: |
+          set -x
+
+          release_charm_to_stable(){
+            charm_name="$1"
+            charm_resource="$2"
+
+            # get the last charm revision number
+            charm_rev=$(charmcraft revisions "${charm_name}" | awk 'FNR == 2 {print $1}')
```

Comment:
when promoting from edge to stable, ideally we would promote the same charm and resource revisions. They would have to match, otherwise it wouldn't be quite the same thing.

Reply:
This is now implemented.